sql/patch_90_91_b.sql

```diff
@@ -0,0 +1,27 @@
+-- Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+-- Copyright [2016-2017] EMBL-European Bioinformatics Institute
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+# patch_90_91_b.sql
+#
+# Title: Add species_id field
+#
+# Description:
+# Add species_id column in the mapping_session table is a possible solution to help identifying the strain with the multiSpecies database mapping sessions.
+
+ALTER TABLE mapping_session ADD COLUMN species_id INT(10) UNSIGNED NOT NULL DEFAULT 1;
```

Comment:
This alter statement should be explicit:
ADD COLUMN species_id INT(10) UNSIGNED AFTER ....
since our data loading from flat files, column order matters. Ensure it's always in the correct order.